src/index.js

```diff
@@ -1,12 +1,26 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
 import App from './components/App'
+import {Provider} from 'react-redux'
+import {createStore} from 'redux'
+import reducer from './reducer'
 import './styles/index.css'
 
 import { Router, Route, hashHistory } from 'react-router'
 
+const initialState = {
+  time: null,
+  amPm: null,
+  dayOfWeek: null,
+  date: null
+}
+
+const store = createStore(reducer, initialState, window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__())
+
 ReactDOM.render((
-  <Router history={hashHistory}>
-    <Route path='/' component={App} />
-  </Router>
+  <Provider store={store}>
+    <Router history={hashHistory}>
```

Comment:
There's a lib for managing history in redux, which makes the URLs much prettier.  If it's not an easy add, please file an issue to add this, otherwise please add.  :)